server/src/config.ts

```diff
@@ -161,3 +162,19 @@ function getTimeZoneOffset(): number {
     return serverUtcOffset;
   }
 }
+
+function getHasticServerUrl() {
+  const urlString = getConfigFieldAndPrintOrExit('HASTIC_SERVER_URL', 'ws://localhost:8002');
+
+  try {
+    const url = new URL(urlString);
+    if (url.protocol !== 'ws:') {
+      throw new Error('Invalid protocol');
+    }
+
+    return url;
+  } catch {
```

Comment:
```suggestion
  } catch {
```

as far as I know, `catch` without argument is only supported in node 10+ or something like that